client/src/body/physics.ts

```diff
@@ -20,11 +18,20 @@ export interface IPhysicsFeature {
     setFactor: (factor: number) => void
 }
 
+function getPhysicsFeature(feature: PhysicsFeature): number {
+    const value = localStorage.getItem(feature)
+    return value ? parseFloat(value) : 1.0
+}
+
+function setPhysicsFeature(feature: PhysicsFeature, factor: number): void {
+    localStorage.setItem(feature, factor.toFixed(3))
+}
```

Comment:
This is still just for testing purposes right?

Reply:
yes, it's not even operative at the moment.  but when we want to have physics constants different for different islands we will have to design a few workable universes with this.